test/controllers/users_controller_test.rb

```diff
@@ -10,7 +10,7 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
 
   describe 'create method' do
     it 'should raise if user is not admin' do
-      User::ROLE.reject { |role| role == :admin }.each do |role|
+      User.role.values.reject { |role| role == :admin }.each do |role|
```

Comment:
Turns out I didn't fully understand this but I was close. It allows you to access classes and constants on modules. SO:

````
module Foo
  class Some
  CONST = 1
     def Thing
     end
   end
end
````
You'd call these things like such:
`Foo::Some`
`Foo::Some::CONST`
`Foo::Some.Thing`

There is a pretty good thing that explains more here: http://www.railstips.org/blog/archives/2009/05/15/include-vs-extend-in-ruby/

This is all of course if you are interested, and didn't know this to begin with. I certainly didn't :)